sdk/src/com/appnexus/opensdk/utils/Settings.java

```diff
@@ -58,6 +58,8 @@
     public Location location = null;
     public int locationDecimalDigits = -1;
 
+    public boolean preventWebViewScrolling = true;
```

Comment:
A setter for this variable must be exposed using `SDKSettings.java`. Can you make those changes and update this Pull request.

Reply:
Hi @avohraa 

I added a getter and a setter here: https://github.com/appnexus/mobile-sdk-android/pull/53/files#diff-8a4ac67006235815c5ccefdcc2c9c49963298184ed9fe6abf391b9891a2487d3

Reply:
Thanks @Gaetan 

I feel that maybe we don't require to set the default value of `preventWebViewScrolling` to `true`. Whomsoever feels like enabling it can use the setter that you have added.

@anwzhang @Kowshickkarthick can confirm on that.

Reply:
I chose that as a default because that's the current behavior. Not sure if it would have a big impact.

Reply:
If that scroll prevention has no purpose, I could just remove it all together. The diff would then only be the removal of the `scrollTo` method in `AdWebView.java`.

Reply:
HI @avohraa setting the default value of preventWebViewScrolling to true is the right behavior.
We need to prevent the scrolling in WebView by default. 

Thanks for the changes @Gaetan. We will include it as a part of our next SDK release in January
Approving this PR its GTG.